### DISCUSSION TITLE
Include html in Input's label

### DISCUSSION THREAD

**Discussion by patrykkalinowski** [carbon-design-system/carbon-components-svelte#1692] [Q&A] [answered]
Hi,

I'd like to include links in my input labels. Currently HTML tags are generated as-is - is it possible to have them parsed?

**Comment by FallonWaagvkp** [comment]
To create a clickable link, you can use HTML markup in your label, such as the <a> tag with the href attribute to specify the link URL. You can then use CSS to style the link as desired.

**Reply by patrykkalinowski** [reply]
<img width="219" alt="Screenshot 2023-03-17 at 16 58 10" src="https://user-images.githubusercontent.com/7796424/225956195-76e03fe1-ec19-48aa-b072-484330055142.png">

Not sure if I'm doing something wrong here?

**Comment by metonym** [comment, marked as answer]
The `labelText` prop only accepts a string, but the `TextInput` also supports a `labelText` slot that you can use to pass HTML or other components:

```svelte
<script>
  import { TextInput, Link } from "carbon-components-svelte";
</script>

<TextInput placeholder="Enter user name...">
	<svelte:fragment slot="labelText">
	  My label with a <Link size="sm" href="https://www.carbondesignsystem.com/">link</Link>
	</svelte:fragment>
</TextInput>
```

[Svelte REPL](https://svelte.dev/repl/6923e2e609714c1aa19d21d370cc159a?version=3.55.0)
